java-frontend/src/main/java/org/sonar/java/se/checks/SECheck.java

```diff
@@ -64,6 +64,10 @@ public void scanFile(JavaFileScannerContext context) {
     issues.clear();
   }
 
+  public void reportIssue(Tree tree, String message) {
+    reportIssue(tree, message, new HashSet<>());
```

Comment:
I was wondering about the same thing, maybe it would be indeed better to change how it is implemented in the superclass, because this is bit fragile